eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -39,6 +39,11 @@ parameters:
       PythonVersion: 'pypy3'
       CoverageArg: '--disablecov'
       RunForPR: false
+    Linux_Python39rc2:
+      OSVmImage: 'ubuntu-18.04'
+      PythonVersion: '3.9.0-rc.2'
```

Comment:
If you specify `'3.9.0-dev'` then it should get the latest prerelease version available (which won't matter so much now, but it should automatically roll over to the final release when it's available)

Reply:
Hmm. I think I am missing something, I don't believe this is the case. The only pre-cached versions of python are GA up to 3.8.5, and the `UsePythonVersion` task does not support download :( 

I am adding download, followed by installing from the actions manifest if possible.